### PR TITLE
Docker updates

### PR DESCRIPTION
**Pull request by peterdelong**
Update docker configuration to use the appropriate rust toolchain. Also update the README

### CODE REVIEW

File: docker/fedora/Dockerfile [hunk 1]
```diff
@@ -21,6 +21,6 @@ RUN yum install -y \
 ENV PATH "/root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```
Comment (shaleh) [inline docker/fedora/Dockerfile]: Is there some way we can read in the `rust-toolchain` so we do not have to constantly update this?

Comment (shaleh) [reply docker/fedora/Dockerfile]: Or do we go to a Dockerfile.in setup?

**Comment by shaleh** [comment]
@peterdelong ??

File: docker/fedora/Dockerfile (hunk 1 above)
Comment (peterdelong) [reply docker/fedora/Dockerfile]: I'm not at all familiar with Docker but the way it's set up right now is that we have a base `Dockerfile.in` that is used to build the other two. It reads in the toolchain version when the build script is run

Comment (shaleh) [reply docker/fedora/Dockerfile]: got it.

**Pull request merged**